ekundayo-ab/hello-books
# #154176246 Implement LF Third Code Review Feedback

### What does this PR do?
Ensures code review feedback are implemented

### Description of Task to be completed?
- Update API documentation to accommodate for the changes made to the API payloads, responses, and entire codebase
- Work on improving on strictly adhering to the style guide, especially in terms of programming patterns, variable and function naming, as well as documentation. Some variables and or functions in your application aren't properly named. For instance variable named in lowercase . Such as "signup". You also have variables that could use better names. Variables such as "book2". Take some time out to go throughout your codebase and make appropriate changes. Let this be your guide http://airbnb.io/javascript/#naming-conventions
- Consider using the appropriate status code in your responses. For instance, I see you using the 404 statuscode for requests that are more pertinent to authentication.

### How should this be manually tested?
1. Pull in latest changes from remote repository to local repository
2. As for other fixes go through codebase to see

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
#153553440 Implement LF third code review feedback

### Screenshots (if appropriate)
N/A

### Questions:
N/A
